extensions/wikia/Email/EmailController.class.php

```diff
@@ -56,6 +56,11 @@ public function init() {
 		try {
 			$this->assertCanAccessController();
 
```

Comment:
I noticed that we go through this init and data checks every time we call ```renderView``` and other dispatching methods.  This was causing an error to be thrown within the ```getBody``` method when it calls ```renderView``` on the ```main``` method.  It still seemed to work though since there's no real code in that method and Nirvana still builds the template out using the data passed.

Adding the ability to disable the init step allows us to skip all this setup when we aren't actually passing in ```currentUser``` et al.